core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -1301,7 +1366,7 @@ private String[] getRoleNamesWithDomain(String sqlStmt, String userName, int ten
      * @throws SQLException
      * @throws UserStoreException
      */
-    protected Connection getDBConnection() throws SQLException, UserStoreException {
+    protected Connection getConnection() throws SQLException, UserStoreException {
```

Comment:
this can be private right?

We don't expect implementers to get the direct DB connection bypassing the circuit breakers

Reply:
@senthalan agreed. let's change it to private.